app/models/form/sales/pages/person1_gender_identity_joint_purchase.rb

```diff
@@ -5,10 +5,12 @@ def initialize(id, hsh, subsection)
     @header = ""
     @description = ""
     @subsection = subsection
-    @depends_on = [{
-      "jointpur" => 1,
-      "details_known_3" => 1,
-    }]
+    @depends_on = [
+      { "hholdcount" => 1, "jointpur" => 1 },
+      { "hholdcount" => 2, "jointpur" => 1 },
+      { "hholdcount" => 3, "jointpur" => 1 },
+      { "hholdcount" => 4, "jointpur" => 1 },
+    ]
```

Comment:
There is currently no `details_known_1` so depending on the household count to conditionally display the question. Might need to change when/if `details_known_1` gets added